calico/about/product-comparison.mdx

```diff
@@ -0,0 +1,87 @@
+| Threat graph                                                 |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Packet capture                                               |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Elasticsearch  logs (flow, l7, audit, bgp, dns, events)      |                          |                          |                          |
+| Alerts                                                       |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Kibana DNS dashboards                                        |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Traffic Flow Visualizer                                      |                          | <center>&#10003</center> | <center>&#10003</center> |
+| **Multi-cluster management**                                 |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Federated identity and services                              |                          | <center>&#10003</center> | <center>&#10003</center> |
+| **Threat defense**                                           |                          |                          |                          |
+| Anomaly detection                                            |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Container threat detection                                   |                          | <center>&#10003</center> |                          |
+| Workload-centric Web Application Firewall (WAF)              |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Honeypods to see intruder activity                           |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Add threatfeeds to trace suspicious network flows            |                          | <center>&#10003</center> | <center>&#10003</center> |
+| **Reports**                                                  |                          |                          |                          |
+| Compliance reports                                           |                          | <center>&#10003</center> | <center>&#10003</center> |
+| CIS benchmark reports                                        |                          | <center>&#10003</center> | <center>&#10003</center> |
+| **Monitor Calico components**                                |                          |                          |                          |
+| Prometheus                                                   | <center>&#10003</center> | <center>&#10003</center> | <center>&#10003</center> |
+| BYO Prometheus                                               |                          | <center>&#10003</center> | <center>&#10003</center> |
```

Comment:
I don't think Prometheus and BYO Prometheus make sense as separate lines. Also, Calico OSS supports BYO Prometheus (we don't package any prometheus with Calico OSS)

Reply:
Deleted BYO row.